common/lib/dependabot/shared_helpers.rb

```diff
@@ -234,13 +205,39 @@ def self.configure_git_credentials(credentials)
           "@#{cred.fetch('host')}"
 
         git_store_content += authenticated_url + "\n"
+        configure_git_to_use_https(cred.fetch('host'))
```

Comment:
```suggestion
        configure_git_to_use_https(cred.fetch("host"))
```

Should fix the rubocop violation